src/DetectChanges/BCBreak/FunctionBased/ParameterDefaultValueChanged.php

```diff
@@ -63,6 +63,7 @@ public function __invoke(
                     continue;
                 }
 
+                /** @infection-ignore-all */
```

Comment:
Should not ignore here: need to dig down to why infection adds mutations here :thinking:

Reply:
The exception wasn't expected in the tests, so it tries to remove the throw.

Reply:
We likely need to see if there's a test scenario to be written here